plugins/amoeba/openmmapi/include/openmm/internal/AmoebaGeneralizedKirkwoodForceImpl.h

```diff
@@ -37,6 +37,8 @@
 #include "openmm/Kernel.h"
 #include <string>
 
+using std::vector;
```

Comment:
Never use the `using` keyword in a header file, only in a cpp file.  It forces it on every file that includes it.